chatcommands.py

```diff
@@ -361,7 +361,10 @@ def blame2(msg, x):
     for i, char in enumerate(reversed(x)):
         user += (len(base)**i) * base[char]
 
-    unlucky_victim = msg._client.get_user(user)
+    try:
+        unlucky_victim = msg._client.get_user(user)
+    except HTTPError:
+        unlucky_victim = msg._client.get_user(random.choice(msg.room.get_current_user_ids()))
```

Comment:
How about making it more amusing and picking the person who invoked the command instead?

Reply:
Sounds like a good idea